### ISSUE TITLE
SS-Backup cannot exit

### ISSUE THREAD

**Issue by smaragdus** [jpt13653903/ss-backup#1]
Today I downloaded [SS-Backup](https://sourceforge.net/projects/ss-backup/) version **1.0** from [SourceForge](https://sourceforge.net/projects/ss-backup/files/) and after I ran the program (`SS-Backup.exe`) I tried to close it but it could not exit, screen:

![SS-Backup 1 0 - 2021-05-10 - exit - 001](https://user-images.githubusercontent.com/9025757/117692513-c3685380-b1c5-11eb-96e2-da5cce6227b6.png)

showing message `Exit Paused`. So I needed to kill the process via a task manager. The exit operation could not be completed neither from **Title Bar->Close Button** nor from **File ->Exit**. I tried several times but I got the same result.

This happens on **Windows 8 x64**.

**Comment by jpt13653903** [comment]
Yes -- Not very intuitive: I should probably change it...

Exiting using the cross in the corner adds the "Exit" task.  You need to run the list (i.e. press the play button) to actually execute the list, thereby running the "Exit" task.

**Comment by jpt13653903** [comment]
On a side note -- I would strongly recommend you using some other backup software.  This was a little thing I wrote a VERY long time ago, and I'm not really supporting it anymore.

**Comment by smaragdus** [comment]
@jpt13653903 

Thanks for the immediate response, for the explanation and for the advice.

Actually I came across this program of yours while browsing your SourceForge [profile](https://sourceforge.net/u/jptaylor/profile/) after I had discovered your [Engineering Calculator](https://sourceforge.net/projects/alwaysontopcalc/) (very nice work by the way). SS-Backup drew my attention because of the nice and clean GUI, the minimal size of the executable and the good language used (C++).

I am closing this issue.

Regards
